src/website/app/Router.js

```diff
@@ -7,6 +7,7 @@ import ComponentDocExample from './ComponentDocExample';
 import Page from './Page';
 import sections from './pages';
 import ComponentDoc from './pages/ComponentDoc';
+import FourOhFour from './pages/FourOhFour';
```

Comment:
How about `PageNotFound` or `FileNotFound`?  I'd also rename the `404.md` file name match as well.